vulnerabilities/importers/mozilla.py

```diff
@@ -0,0 +1,180 @@
+import re
+from typing import List
+from typing import Set
+
+import yaml
+from bs4 import BeautifulSoup
+from markdown import markdown
+from packageurl import PackageURL
+
+from vulnerabilities.data_source import Advisory
+from vulnerabilities.data_source import GitDataSource
+from vulnerabilities.data_source import Reference
+from vulnerabilities.data_source import VulnerabilitySeverity
+from vulnerabilities.helpers import is_cve
+from vulnerabilities.helpers import split_markdown_front_matter
+from vulnerabilities.severity_systems import scoring_systems
+
+REPOSITORY = "mozilla/foundation-security-advisories"
+MFSA_FILENAME_RE = re.compile(r"mfsa(\d{4}-\d{2,3})\.(md|yml)$")
```

Comment:
IMHO this should be made as callable closing on .search or .match with an explicit name. And should use named groups for clarity.

Reply:
Is this better ? 
``search_mfsa_in_filename = re.compile(r"mfsa(\d{4}-\d{2,3})\.(md|yml)$").search``
There is no grouping, only mfsa id is returned so I've avoided named groups.

Reply:
:+1: